pypeit/scripts/setup_coadd2d.py

```diff
@@ -178,7 +186,12 @@ def main(args):
                 if args.weights is None else args.weights
         cfg['coadd2d']['spat_toler'] = par['coadd2d']['spat_toler'] \
                 if args.spat_toler is None else args.spat_toler
-
+        cfg['coadd2d']['spec_samp_fact'] = par['coadd2d']['spec_samp_fact'] \
+                if args.spec_samp_fact is None else args.spec_samp_fact
+        cfg['coadd2d']['spat_samp_fact'] = par['coadd2d']['spat_samp_fact'] \
+                if args.spat_samp_fact is None else args.spat_samp_fact
+        
+        # TODO JFH Why are exclude_slits and only_slits set here when they are parameters in the parset?
```

Comment:
Good question.  Assuming I was the one that wrote this, my guess is that I was mimicking the fact that `only_lists` and `exclude_slits` are keyword arguments of `CoAdd2D`.  I'm fine with having them removed as explicit keyword arguments and included as part of `cfg` instead.

Reply:
Do we want to deal with this now?

Reply:
No, we'll leave for another PR.